src/core/credentials/transport/google_default/google_default_credentials.cc

```diff
@@ -398,43 +399,60 @@ class GoogleDefaultCallCredentialsWrapper : public grpc_call_credentials {
   grpc_core::RefCountedPtr<grpc_call_credentials> alts_credentials_;
 };
 
-static grpc_core::RefCountedPtr<grpc_call_credentials> make_default_call_creds(
-    grpc_error_handle* error) {
+absl::StatusOr<std::pair<grpc_core::RefCountedPtr<grpc_call_credentials>,
+                         DefaultCallCredentialsCreationMethod>>
+CreateGoogleDefaultCallCredentials() {
```

Comment:
This function and the `GoogleDefaultCallCredentialsWrapper` should both be in an anonymous namespace, since they're not needed external to this file.